src/crypto/index.js

```diff
@@ -228,12 +228,24 @@ utils.inherits(Crypto, EventEmitter);
  * Initialise the crypto module so that it is ready for use
  *
  * Returns a promise which resolves once the crypto module is ready for use.
+ *
+ * @param {Object} kwargs keyword arguments.
+ * @param {string} kwargs.exportedOlmDevice (Optional) data from exported device
+ *     that must be re-created.
  */
-Crypto.prototype.init = async function() {
+Crypto.prototype.init = async function(kwargs) {
```

Comment:
Arguments like this are generally called `opts` in other functions.

```suggestion
Crypto.prototype.init = async function(opts) {
```

(with the change applied elsewhere in the function too, obviously)

Reply:
Ok, done.

To me `opts` can be misleading as it sounds like “optional arguments” while some may in fact be required. Usage of `kwargs` come from Python world.

Although I agree that naming consistency is more important, usage of `kwargs` instead of `opts` should be discussed in a separate issue/PR, if at all.